src/test/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -28,9 +29,10 @@ public void testFormatStripLatexCommands() {
     @Test
     public void testFormatTextit() {
         // See #1464
-        assertEquals("text", formatter.format("\\textit{text}"));
+        assertEquals("\uD835\uDC61\uD835\uDC52\uD835\uDC65\uD835\uDC61", formatter.format("\\textit{text}"));
```

Comment:
![italics](https://cloud.githubusercontent.com/assets/676652/22789274/c6353a6a-eee2-11e6-9d42-5642a83c3f5b.png)

Okay... I really kinda works - Although is looks strange ;-)